### PR TITLE
Add links to PCP+Grafana dashboards

### PR DESCRIPTION
**Pull request by zmc**
Signed-off-by: Zack Cerza <zack@redhat.com>

### CODE REVIEW

**Comment by zmc** [comment]
Depends on https://github.com/ceph/paddles/pull/69

File: pulpito/templates/job.html [hunk 1]
```diff
@@ -77,6 +77,14 @@ <h1 id="tables"><a href="/{{ job.name }}/">{{ job.name }}</a><span class='spaced
                     </a>
                 </p>
               {% endif %}
+              {% if job.pcp_grafana_url %}
+                <p>
+                    PCP+Grafana:
```
Comment (dmick) [inline pulpito/templates/job.html]: Perhaps better to make the string displayed to users a little more rube-friendly (like, "Perf Graphs" or something)?

Comment (zmc) [inline pulpito/templates/job.html]: Sure, that's probably a good idea. I'll update soon

**Comment by zmc** [comment]
@dmick updated

**Pull request merged**